release.yaml

```diff
@@ -30,8 +30,10 @@ rancher-backup-crd:
 - 102.0.0+up3.1.0-rc1
 rancher-cis-benchmark:
 - 3.0.1-rc6
+- 3.1.0-rc1
```

Comment:
since the last released version is `3.0.0` (ref: https://github.com/rancher/charts/tree/release-v2.7/charts/rancher-cis-benchmark), following our pattern of bumping the major version for all charts with PSP changes we should be releasing `4.0.0` with this release.

Also `3.0.1-rc6` should have been `make remove`d as well; can we remove that chart in this PR as well from the `release.yaml` and `charts`/`assets`/`index.yaml`?

Reply:
Sure thing, will make the changes!

Reply:
This is updated!